config/database.php

```diff
@@ -36,7 +36,7 @@
         'sqlite' => [
             'driver' => 'sqlite',
             'database' => env('DB_DATABASE', database_path('database.sqlite')),
-            'prefix' => env('DB_PREFIX'),
+            'prefix' => env('DB_PREFIX', ''),
```

Comment:
For all 3 instances of this issue in this file:
It's better to have a default value.
Otherwise users will have to put `DB_PREFIX=` into their `.env` file.
Users won't always know this, as this is not immediately obvious.